plugins/inputs/vsphere/README.md

```diff
@@ -532,6 +542,220 @@ For a detailed list of commonly available metrics, please refer to
 * virtualDisk stats for VM
   * disk (name of virtual disk)
 
+# VMware vSphere Input Plugin - vSAN extension
+
+vSAN resource is a special type of resource that can be collected by the plugin.
+The configuration of vSAN resource is slightly different from hosts, vms and other resources.
+
+## Prerequisites
```

Comment:
Can you please integrate this more with the existing text? This feels a bit alien here especially when using a first-level heading which reads like this is docu for another plugin.